platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/Migratable.java

```diff
@@ -34,7 +36,7 @@
  * </b>
  * </p>
  */
-public interface Migratable {
+public interface Migratable extends Describable {
```

Comment:
Can't we remove getDescription() from Migratable now that it extends Describable?